apps/e2ee-server/src/utils/hexUtils.ts

```
@@ -0,0 +1,30 @@
+const stripHexPrefix = (str: string) =>
+  hasHexPrefix(str) ? str.slice(2) : str;
+
+const addHexPrefix = (str: string): `0x${string}` =>
+  hasHexPrefix(str) ? (str as `0x${string}`) : `0x${str}`;
+
+function hexStringToUtf8String(hexString: string): string {
+  const hex = hexString.replace('0x', '');
+
+  try {
+    const bytes = new Uint8Array(
+      hex.match(/.{1,2}/g)?.map((byte) => parseInt(byte, 16)) || [],
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)